blog/2020-10-08-debezium-community-stories-with-renato-mefi.adoc

```diff
@@ -0,0 +1,143 @@
+In today's edition it's my pleasure to talk to https://twitter.com/renatomefi[Renato Mefi], a long-time Debezium user and contributor.
+--
+
+[.centered-image.responsive-image]
+====
+++++
+<img src="/images/renatomefi.jpg" style="max-width:50%;" class="responsive-image">
+++++
+====
+
+*Renato, could you introduce yourself? What is your job, if you're not contributing to Debezium?*
+
+Hello all, I'm Renato and my first Debezium commit was on Nov 12, 2018, it's been a long and fun ride so far, and I'm glad to have the opportunity to share my story here with you all!
+
+I'm a Staff Software Engineer at https://www.surveymonkey.com/[SurveyMonkey] in Amsterdam, The Netherlands, within the Platform team for our CX (customer experience) suite, if you're curious about what that is, you can https://usabilla.com/blog/introducing-the-getfeedback-suite/[check it out here].
+
+On the internet you're going to find me talking about Docker, Debezium, Kafka, Microservices and other things that I enjoy.
+Although those amazing engineering pieces really excite me, at this moment I'm also really passionate about Platform Engineering teams and how they can operate in an organization, the stories I'm going to tell below represent my view of it, of how critical the role of a platform team can be when adopting new technologies and solving difficult problems for the whole, in this case powered by Debezium!
+
+*What are your use cases for Debezium and CDC in your current project?*
+
+It's a long and enjoyable story (long in terms of the internet), we've been using Debezium since Q4 2018, it's been 2 years at the moment I'm writing those answers here.
+
+When I classify Debezium within our product, I say it is an architectural component, the idea behind this is to position it as a platform/infrastructure concern, in a way that it can reach multiple parts of the stack and services. I consider this abstraction of Debezium one of the key success factors it had in its adoption and growth within our platform, let me explain this better!
+
+Our first use case is likely to be one of the most common ones for CDC, the https://martinfowler.com/bliki/StranglerFigApplication.html[strangler pattern], which for us came before Debezium; so let me tell this part of the story first: when I joined Usabilla (later acquired by SurveyMoney), there was already an effort to move our platform to a new architecture and the strangler pattern was already there, when the first couple of services started to grow, their primary way to bring data out of legacy was to poll the database, and needless to say, this could go very wrong! Our legacy database is a MongoDB cluster, and since I was preoccupied with the polling approach I started to dig into possibilities, I was hoping to find something like a streaming api for it, but what I ended up encountering was the database changelog, "oplog" as it's called in Mongo!
```

Comment:
```suggestion
Our first use case is likely to be one of the most common ones for CDC, the https://martinfowler.com/bliki/StranglerFigApplication.html[strangler pattern], which for us came before Debezium; so let me tell this part of the story first: when I joined Usabilla (later acquired by SurveyMoney), there was already an effort to move our platform to a new architecture and the strangler pattern was already there, when the first couple of services started to grow, their primary way to bring data out of legacy was to poll the database, and needless to say, this could go very wrong! Our legacy database is a MongoDB cluster, and since I was preoccupied with the polling approach I started to dig into possibilities, I was hoping to find something like a streaming api for it, but what I ended up encountering was the database changelog (link:https://en.wikipedia.org/wiki/Write-ahead_logging[Write-ahead logging], "oplog" as it's called in Mongo!
```